File: projet/source/f1levier/app/src/main/java/fr/utbm/lo52/f1levier/db/entity/Team.java
package fr.utbm.lo52.f1levier.db.entity;


import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.ForeignKey;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "team",
        foreignKeys = { @ForeignKey(entity = Race.class,
                parentColumns = "id",
                childColumns = "race_id")})
public class Team {

    @PrimaryKey(autoGenerate = true)
    public int id;

    public String name;

    @ColumnInfo(name="race_id")
    public int raceId;

    public Team(String name, int raceId) {
        this.name = name;
        this.raceId = raceId;
    }
}

File: Projet/app/app/src/main/java/com/codep25/codep25/model/entity/Team.kt
package com.codep25.codep25.model.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(
    tableName = "teams"
)
data class Team(
    @PrimaryKey(autoGenerate = true)
    val id_team: Long,

    @ColumnInfo(name = "team_name")
    val name: String
)